pipeline/extract/document.py

```diff
@@ -89,6 +89,32 @@ def to_string(self) -> str:
 
         return "\n".join(page_text)
 
+    def flip_y_coordinates(self) -> "Page":
```

Comment:
It looks like the Adobe extract API gives us these coordinates in the format we need to begin with, and given that we're using that format in the presentation layer too, maybe it's worth updating the embedded extractor code to produce the format we need for presentation rather than this additional flipping?

Reply:
You're right, good spot. For some reason I thought I'd made the mistake of putting the coordinate transformation in a bit of code that meant we'd have to rerun the Adobe API calls, but it's not. Will open a new PR with those changes next week.